admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/ZoneUserServiceImpl.java

```diff
@@ -309,7 +309,7 @@ public StatusResponseDto updateZoneUserMapping(String userId, boolean isActive)
 				throw new MasterDataServiceException(ZoneUserErrorCode.USER_MAPPING_EXIST.getErrorCode(),
 						ZoneUserErrorCode.USER_MAPPING_EXIST.getErrorMessage());
 			}
-			masterdataCreationUtil.updateMasterDataStatus(ZoneUser.class, userId, isActive, "userId");
+			masterdataCreationUtil.updateMasterDataStatus(ZoneUser.class, userId.toLowerCase(), isActive, "userId");

```

Comment:
When lowercase conversion is taken care of in the queries. why are we converting to lowercase here?

Reply:
We can't do those changes in release script anusha, as we might have data redundancy which can lead to errors.